### PR TITLE
Corrected spelling error of the documentation of BluetoothClient: frue → true 

### PR DESCRIPTION
**Pull request by GordonL0049**
Corrected issue [#2938](https://github.com/mit-cml/appinventor-sources/issues/2938) in this PR. Changes:

- Correct the typo from `frue` to `true` in two files.

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by ewpatton** [comment]
@GordonL0049 Note that the HTML is built from the Markdown versions of the files. You need to make sure that the Markdown also reflects the fix.

**Review by ewpatton — CHANGES_REQUESTED** [review]
Once the Markdown file has been updated this should be good to go.

**Comment by GordonL0049** [comment]
Done, but when I review the changes, I found myself accidentally `Merge branch 'mit-cml:master' into master` ... is it OK to leave it there? If not, is there a way for me to remove it?

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java [hunk 1]
```diff
@@ -931,10 +931,9 @@ private void setBlockSubsetProperty(String asJson) {
       editor.getProjectEditor().changeProjectSettingsProperty(
           SettingsConstants.PROJECT_YOUNG_ANDROID_SETTINGS,
           SettingsConstants.YOUNG_ANDROID_SETTINGS_BLOCK_SUBSET, asJson);
-    }
-    
-    if (editor.isLoadComplete()) {
-      ((YaFormEditor)editor).reloadComponentPalette(asJson);
+      if (editor.isLoadComplete()) {
+        ((YaFormEditor)editor).reloadComponentPalette(asJson);
+      }
```
Comment (ewpatton) [inline appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java:936]: Revert this change.

**Comment by ewpatton** [comment]
I would recommend in the future doing your development on a branch started from master (or ucr, as appropriate), rather than on the branch directly. When we merge a change, we will create a single logical commit using a squash merge into the base branch. If you do your development directly on base branch this will cause your copy to end up out of sync with the MIT copy.

**Comment by GordonL0049** [comment]
Already updated. Does it look OK?

**Review by ewpatton — APPROVED** [review]

**Pull request merged**